### PR TITLE
[develop] Adding clean_old_jobs to pgjsonb returner for Fluorine

### PR DESCRIPTION
**Pull request by garethgreenaway**
### What does this PR do?
Adding the clean_old_jobs function the pgjsonb returner for Fluorine

### What issues does this PR fix or reference?
#47316 

### Commits signed with GPG?
Yes

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/latest/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

File: salt/returners/pgjsonb.py [hunk 1]
```diff
@@ -59,6 +59,22 @@
     alternative.pgjsonb.ssl_cert: '/etc/pki/mysql/certs/localhost.crt'
     alternative.pgjsonb.ssl_key: '/etc/pki/mysql/certs/localhost.key'
 
+Should you wish the returner data to be cleaned out every so often, set
+`keep_jobs` to the number of hours for the jobs to live in the tables.
```
Comment (rallytime) [inline salt/returners/pgjsonb.py]: I know this is very picky, but these instances using single back-ticks throughout these new docs need to be using double back-ticks in order for our docs to display these terms as `code-format`. (Markdown uses single or double, RST uses double-only.)

File: salt/returners/pgjsonb.py [hunk 2]
```diff
@@ -59,6 +59,22 @@
...
+set `archive_jobs` to True.  Salt will create 3 archive tables
+
+- `jids_archive`
+- `salt_returns_archive`
+- `salt_events_archive`
+
+and move the contents of `jids`, `salt_returns`, and `salt_events` that are
+more than `keep_jobs` hours old to these tables.
+
+.. versionadded:: 2015.5.0
```
Comment (rallytime) [inline salt/returners/pgjsonb.py]: `Fluorine`?

**Review by rallytime — CHANGES_REQUESTED** [review]
LGTM, with the exception of a couple of very small comments.

Can you write some tests for this as well?

**Review by cro — APPROVED** [review]

**Review by rallytime — APPROVED** [review]

**Comment by rallytime** [comment]
re-run py

**Pull request merged**